index.html

```diff
@@ -59,6 +68,7 @@ <h1>Your Options (Alternatives)</h1>
 
         <section id="pairwise_criteria" class="pairwise_criteria pairwisetable">
           <h1>Pairwise Comparisons For Your Criteria With Respect To Your Goal</h1>
+          <h3 id="pairwise_criteria_sentence"></h3>
```

Comment:
let's use an `<aside>` here instead of H3. More semantically appropriate, and easier to style it since it won't interfere with other H3 styles on the page